src/Microsoft.Dnx.Runtime.Internals/RuntimeEnvironmentExtensions.cs

```diff
@@ -27,5 +30,45 @@ public static string GetShortVersion(this IRuntimeEnvironment env)
         {
             return $"{env.RuntimeType}-{env.RuntimeArchitecture}-{env.RuntimeVersion}";
         }
+
+        public static string GetRuntimeIdentifier(this IRuntimeEnvironment env)
+        {
+            if (string.Equals(env.RuntimeType, RuntimeTypes.Mono, StringComparison.Ordinal))
+            {
+                return "mono";
+            }
+
+            string os = env.OperatingSystem;
+            string ver = env.OperatingSystemVersion;
+            if (string.Equals(os, RuntimeOperatingSystems.Windows, StringComparison.Ordinal))
+            {
+                os = "win";
+
+                // Convert 6.x to the correct branding version for the RID
+                var parsedVersion = Version.Parse(ver);
+                if(parsedVersion.Major == 6 && parsedVersion.Minor == 1)
```

Comment:
:+1:

Removing the manifest still allows me to proceed with the rest of the changes (since I can still use win7/win8/linux/darwin as pivots, just not win81 and win10), so I think I'll do that just to ensure we don't mess things up too much. We just need to ensure #1970 handles it :)